06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable */
```

Comment:
Did you review the AirBnB and Andela JavaScript style guides? Please fix these violations.